lib/asciidoctor/converter/semantic_html5.rb

```diff
@@ -80,6 +224,63 @@ def convert_inline_anchor node
     end
   end
 
+  def convert_inline_quoted node
+    attributes = common_html_attributes node.id, node.role
+    case node.type
+    when :strong
+      %(<strong#{attributes}>#{node.text}</strong>)
+    when :emphasis
+      %(<em#{attributes}>#{node.text}</em>)
+    when :monospaced
+      %(<code#{attributes}>#{node.text}</code>)
+    when :mark
+      %(<mark#{attributes}>#{node.text}</mark>)
+    when :superscript
+      %(<sup#{attributes}>#{node.text}</sup>)
+    when :subscript
+      %(<sub#{attributes}>#{node.text}</sub>)
+    when :single
+      attributes = common_html_attributes node.id, node.role, 'single_quote'
+      %(<span#{attributes}>&#8216;#{node.text}&#8217;</span>)
+    when :double
+      attributes = common_html_attributes node.id, node.role, 'double_quote'
+      %(<span#{attributes}>&#8220;#{node.text}&#8221;</span>)
+    else
+      %(<span#{attributes}>#{node.text}</span>)
+    end
+  end
+
+  def convert_inline_break node
+    %(#{node.text}<br />)
+  end
+
+  def convert_inline_kbd node
+    if (keys = node.attr 'keys').size == 1
+      %(<kbd>#{keys[0]}</kbd>)
+    else
+      %(<span class="keyseq"><kbd>#{keys.join '</kbd>+<kbd>'}</kbd></span>)
+    end
+  end
+
+  def convert_inline_button node
+    %(<samp class="button">#{node.text}</samp>)
```

Comment:
I don't think `<samp>` is the right element to represent a button. MDN says:

> The [\<samp>](https://developer.mozilla.org/en-US/docs/Web/HTML) element is used to enclose inline text which represents sample (or quoted) output from a computer program.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/samp

Having said that, I don't have a good alternative.
Currently, we are using `<b>`... I guess we could use `<span>`?

Reply:
Yea, this one is tough, `samp` wasn't perfect, and I think `b` matches a bit less. `span class="button"` or something might be the "right" solution.